### PR TITLE
Use the same llvm's version as crystal-lang package for CI's darwin build

### PR DESCRIPTION
**Pull request by bew**
WIP I want to see if it fails, or if we need to change something else..

I don't know a lot in macos/darwin stuff so if you know what to do, please fix it :)

### CODE REVIEW

**Comment by bew** [comment]
passed
:stuck_out_tongue: (I'm so happy in my head)

File: bin/ci [hunk 1]
```diff
@@ -84,7 +84,10 @@ prepare_build() {
 
   on_linux docker pull "jhass/crystal-build-$ARCH"
 
-  on_osx brew install llvm crystal-lang
+  on_osx brew install llvm@5 crystal-lang
```
Comment (matiasgarciaisaia) [inline bin/ci]: `crystal-lang` itself currently depends on `llvm@5`, so it's not necessary to `install` it.

We do need to link it, though - we can get the currently depended `llvm` version via `jq` with `brew info --json=v1 crystal-lang | jq '.[].dependencies | .[] | select(startswith("llvm"))'`. Not sure if overkill, but avoiding re-specifying the LLVM version we need seems sweet.

Comment (bew) [reply bin/ci]: nice one, is there `jq` installed already?

Comment (bew) [reply bin/ci]: It worked perfectly ;)

**Comment by bew** [comment]
All clear!

**Pull request merged**